.github/workflows/lambda-do-release-runners.yml

```diff
@@ -23,6 +23,21 @@ jobs:
         with:
           ref: ${{ inputs.tag }}
 
+      - name: Install dependencies - Runner Binaries Syncer
```

Comment:
I believe you'll also want to update the release-action (line 73 of this file) to include this new zip

Reply:
Good catch! Fixed!